### ISSUE TITLE
Sign in WebView's lost its state after rotation

### ISSUE THREAD

**Issue by littlecd** [fan123199/v2ex-simple#16]
Open the WebView for sign in. Then randomly click several links and rotate your device. You will see the web page refreshes back to the sign in page.

Before rotation

<img width="315" alt="Screen Shot 2023-09-10 at 9 39 57 PM" src="https://github.com/fan123199/v2ex-simple/assets/144590216/25906faf-419a-42b2-b38a-08a4163f14b9">

After rotation
<img width="632" alt="Screen Shot 2023-09-10 at 9 40 04 PM" src="https://github.com/fan123199/v2ex-simple/assets/144590216/3d5409e7-a52d-4b67-9012-71b3fb0577b2">

**Comment by fan123199** [comment]
Yeah, it is the problem. I do not consider the rotation scenario when coding.  It maybe fixed in next version.
